sub/source/docs/sub-hardware.rst

```diff
@@ -0,0 +1,83 @@
+.. _sub-hardware:
+
+==================================
+Default Sub Hardware Configuration
+==================================
+
+While the Sub firmware is currently being generalized to allow more flexibility, such as RC control, re-arrangement of outputs and inputs, etc.,
+current code is targeted for a specific ROV configuration with the following hardware setup:
+
+- Control is via a surface tether from a ground control station via MAVLink commands to the Sub's autopilot telemetry port, usually employing a joystick and multiple buttons for mode changes and peripheral hardware control. Control functions and autopilot outputs for thrusters and peripheral hardware have specific defaults (see :ref:`Sub defaults <sub-defaults>` below)
+- A depth sensor (underwater barometer) is required.
+- Frame configuration is defaulted to Vectored with 6 thrusters.
+
+.. _sub-defaults:
+
+Sub Defaults
+============
+
+* Default :ref:`frame type <FRAME_TYPE>` is VECTORED
+* RC is disabled via :ref:`RC_PROTOCOLS<RC_PROTOCOLS>` = 0 and :ref:`FLTMODE_CH<FLTMODE_CH>` = 0.
+
+MAVLink Overrides
+-----------------
+Since pilot directional control, normally provided by RC in other vehicles is not enabled by default. Sub uses RC overrides via MAVLink, and the channel mapping is fixed:
+
+===============  ===================
+Pilot control    RC channel override
+===============  ===================
+Roll             1
+Pitch            0
+Yaw              3
+Throttle(Depth)  2
+Forward          4
+Lateral          5
+Camera Pan       6 (via AUX function on RC7)
+Camera Tilt      7 (via AUX function on RC8)
+Lights1          8
+Lights2          9
+Video Switch     10
+===============  ===================
+
+Output Assignments
+------------------
+The default output assignments are:
+
+======       ========
+Output       Function
+======       ========
+1            Motor1
+2            Motor2
+3            Motor3
+4            Motor4
+5            Motor5
+6            Motor6
+9            RCin9
+10           Mount1 Pitch
+======       ========
+
+Button Assignments
+------------------
+
+Default Sub uses GCS joystick buttons to implement mode switching, light activation, etc.
+
+======      ========
+Button      Function
+======      ========
+1           Mode Manual
+2           Mode Depth_Hold
+3           Mode Stabilize
+4           Disarm
+5           Shift Function (when pressed buttons provide alternate second function)
+6           Arm
+7           Center Mount1
+8           Joystick Input Hold Set
+9           Tilt Mount1 Down/Pan Mount Left
+10          Tilt Mount1 Up/Pan Mount Right
+11          Joystick Sensitivity Gain Increase/ Trim Pitch Decrease
+12          Joystick Sensitivity Gain Decrease/ Trim Pitch Increase
+13          Lights1 Dimmer/ Trim Roll Decrease
+14          Lights1 Brighter/ Trim Roll Increase
```

Comment:
```suggestion
9           Tilt Mount1 Down / Pan Mount1 Left
10          Tilt Mount1 Up / Pan Mount1 Right
11          Joystick Sensitivity Gain Increase / Trim Pitch Decrease
12          Joystick Sensitivity Gain Decrease / Trim Pitch Increase
13          Lights1 Dimmer / Trim Roll Decrease
14          Lights1 Brighter / Trim Roll Increase
```

Reply:
actually we are both wrong...there should be no white space around the slash when it indicates an alternative....will correct

Reply:
Is that an ArduPilot convention?

I read spacing around slashes as "word/alternative" / "option alternative".

Reply:
its a general grammar/punctuation rule...Goggle "whitespace around slashes":

"Generally, whitespace is not used around slashes (/) when they separate single words or letters. However, a space is typically used before and after the slash when it separates phrases, sentences, or a single word from a phrase, especially to improve readability. 
"

Reply:
That's exactly what I was trying to suggest 😅 

As an example, it should be "Tilt Mount1 Down" / "Pan Mount1 Left" (i.e. two phrases being separated). If it's separating just the words in the middle (e.g. "Tilt Mount1 Down/Pan Mount1 Left") then it comes across as those words being the differentiator of two broader options (e.g. "Tilt Mount1 Down Mount1 Left" / "Tilt Mount1 Pan Mount1 Left").

As a stripped down example, "Lights 1/2" makes sense, and "expands" to "Lights 1 / Lights 2", but "Lights 1/Lights 2" is hard to make sense of.

Reply:
yep, I see your point...will fix